plugin/system_copy.vim

```diff
@@ -83,14 +95,27 @@ endfunction
 xnoremap <silent> <Plug>SystemCopy :<C-U>call <SID>system_copy(visualmode(),visualmode() ==# 'V' ? 1 : 0)<CR>
 nnoremap <silent> <Plug>SystemCopy :<C-U>set opfunc=<SID>system_copy<CR>g@
 nnoremap <silent> <Plug>SystemCopyLine :<C-U>set opfunc=<SID>system_copy<Bar>exe 'norm! 'v:count1.'g@_'<CR>
-nnoremap <silent> <Plug>SystemPaste :<C-U>call <SID>system_paste()<CR>
+nnoremap <silent> <Plug>SystemPaste :<C-U>call <SID>system_paste('below')<CR>
+nnoremap <silent> <Plug>SystemPasteInline :<C-U>call <SID>system_paste('inline')<CR>
```

Comment:
Would it be possible to pull out the addition of the `SystemPasteInline` mapping to another PR? I'd prefer to keep this one focused on the options for the copy and paste commands.